### PR TITLE
Adding Issue Template and Pull Request file

### PR DESCRIPTION
**Pull request by afreenabegum**
@siddhi-244 ,Added issue and Pr template. Once review my changes which fixes issue #2

### CODE REVIEW

**Comment by siddhi-244** [comment]
Please link the pr to issue number

**Review by siddhi-244 — CHANGES_REQUESTED** [review]
Check the file names. It should be same as given in the reference

**Comment by afreenabegum** [comment]
> Check the file names. It should be same as given in the reference

Did I made required changes?please review.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+Remove this line and tell us how the code was tested.
+
+## Checklist:
+Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:
+
+-   [ ] Have you followed the [Contribution Guidelines]()while contributing.
+-   [ ] Have you checked there aren't other open [Pull Requests](https://github.com/BugBustersCommunity/website/pulls) for the same update/change?
+-   [ ] Have you made corresponding changes to the documentation?
+-   [ ] Have you tested the code before submission?
+-   [ ] Have you formatted your code ? (You can use any html,css beautifier)
```
Comment (Aditi-Singh16) [inline .github/PULL_REQUEST_TEMPLATE.md]: @afreenabegum , don't make it interrogative
Instead use simple sentences
change all `Have you` type question to `I have...`
eg -> Have you followed the guidelines? to` I have followed the guidelines.`

**Review by Aditi-Singh16 — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (afreenabegum) [reply .github/PULL_REQUEST_TEMPLATE.md]: > @afreenabegum , don't make it interrogative Instead use simple sentences change all `Have you` type question to `I have...` eg -> Have you followed the guidelines? to` I have followed the guidelines.`

done changes, Please do review.

**Review by siddhi-244 — CHANGES_REQUESTED** [review]
It's I have formatted the code and not I have formatted your code !

**Review by Aditi-Singh16 — APPROVED** [review]

**Review by siddhi-244 — APPROVED** [review]
LGTM

**Pull request merged**